lib/graphql/adapters/base_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "active_job"
+
+module GraphQL
+  module Adapters
+    class BaseJob < ActiveJob::Base
+      DEFAULT_QUEUE_NAME = :default
+
+      queue_as { GraphQL::AnyCable.config.async_broadcasting["queue"] || DEFAULT_QUEUE_NAME }
```

Comment:
The default name must be define in the configuration, no need to use a fallback here:

```suggestion
      queue_as do
        AnyCable.config.active_job_queue_name
      end
```

Reply:
Or, we go with adapter instances, we can specify the queue right when calling `#perform_later`:

```ruby
TriggerJob.set(queue: queue).perform_later(...)
```

Reply:
Thanks!
Done